src/truffle/approve-tokens.js

```diff
@@ -31,6 +34,11 @@ var argv = require('yargs')
     default: 'development',
     describe: 'One of the ethereum networks defined in truffle config'
   })
+  .option('whitelist', {
+    type: 'string',
+    default: 'DutchExchangeProxy',
```

Comment:
I see what you did here, I would prefer if this argument is optional, null or undefined by default, instead of mixin a contract name with an address.

If it's not set, it'll use DutchExchangeProxy as the default address

Reply:
So would you like `--whitelist` to be an address only? No contract name?

Reply:
Exactly